src/scss/constants.scss

```diff
@@ -1,11 +1,19 @@
 $primaryColor: var(--primary-color);
 $backgroundColor: var(--background-color);
 
+$primary--lightest: #a179ff;
+$primary--light: #9669ff;
 $primary: #6f43ff;
-$dimmer-primary: #3f247d;
+$dimmer--darker: #3f247d;
 
+$secondary--lightest: #868686;
+$secondary--light: #6a6a6a;
+$secondary: #333035;
+$secondary--dark: #19181a;
+$secondary--darkest: #090909;
+
+// @debt replace all greyish colors with $secondary alternatives
 $dark-grey: #1a1d23;
-$intermediate-grey: #333035;
 $saturated-black: #19181a;
 $saturated-black-alt: #262427;
 $lighter-intermediate-grey: #3d3a3f;
```

Comment:
Some of these seem to be the same was what you defined above? At the very least they should be aliasing the variable rather than redefining it from scratch?

Reply:
Done in https://github.com/sparkletown/sparkle/pull/1374/commits/c8ed35ee719ea1855d7b0afef9ad5b619666efc1